client/src/main/java/org/apache/celeborn/client/read/CelebornInputStream.java

```diff
@@ -680,6 +681,9 @@ private boolean fillBuffer() throws IOException {
           }
         }
         throw ioe;
```

Comment:
how about `logger.error("....", ioe)`?

Reply:
Because there is a `reportShuffleFetchFailure` RPC call, it may fail, so the log is still output when an exception occurs at the beginning.